### DISCUSSION TITLE
**Question About the Feasibility of This Schematic**

### DISCUSSION THREAD

**Discussion by pilou33620** [ra3xdh/qucs_s#1011] [Q&A] [answered]
Hello everyone,

I am trying to simulate crosstalk with this setup. But unfortunately, it doesn't work: my signals remain at 0 V in my diagram. So I wonder if my setup is correct or if the **coupled microstrip line** does not allow my signals to pass through. If anyone has any ideas, I would appreciate it.

Thank you in advance.

![Capture d’écran 2024-10-21 162704](https://github.com/user-attachments/assets/c74c48cc-a86c-4d80-8924-a60f4009a0e9)

**Comment by ra3xdh** [comment, marked as answer]
The coupled microstrip has no transient model in both old Qucsator and Qucsator RF. See https://github.com/ra3xdh/qucsator_rf/issues/7 and https://github.com/ra3xdh/qucs_s/discussions/681 for more explanations. The adding of the TRAN model for transmission lines is on the way, but I cannot say when the works will be finished.  Help is highly appreciated here.

**Comment by pilou33620** [comment]
Thank you for your response. Aside from providing you with a link to the equations, I do not have the skills to be of much additional help.
Link : https://qucs.sourceforge.net/tech/node77.html

**Comment by tomhajjar** [comment]
Qucs transmission line equations are frequency domain only. 

Also for Time Domain analysis of PCB's, the models would need to account for "coupling" between Edgewise and Broad-Side Microstrip and Stripline traces on other layers as well as vias...

ngspice supports Twisted pair and KSPICE Coupled Multiconductor Line (CPL). You could experiment making sub-circuits with User selectable parameters.

I believe Qucsator Coupled line and Twisted pair support Time Domain.

![2024-10-21_211706](https://github.com/user-attachments/assets/2b02bff1-46e9-498e-b0f3-75c5105ef85e)

**Comment by felix-salfelder** [comment]
It's been on the Qucs [roadmap](https://github.com/Qucs/qucs/blob/develop/ROADMAP) for a while. There is no reason not try this with Qucs-S. I'm a little short on time hunting down heavier stuff. But I can help.
